### PR TITLE
Fix #30 Refactoring for better readability 

### LINKED ISSUE #30: Refactoring to improve readability
We did some amount of clean up during integration of tracey broker 1.1.0
There are more improvements that could be done:
- align naming of variables for better readability
- rename tracey host to rabbitmq host in Jenkins UI

### PR DESCRIPTION
**Pull request by alexsedova**

### CODE REVIEW

File: src/main/java/org/jenkinsci/tracey/RabbitMQTrigger.java [hunk 1]
```diff
@@ -102,7 +101,7 @@ private UsernamePasswordCredentials getCredentials(TraceyHost h, Job<?,?> j) {
     }
 
     private TraceyRabbitMQBrokerImpl configureBroker(Job<?,?> proj, String hid) {
-        TraceyHost th = TraceyGlobalConfig.getById(hid);
+        RabbitMQHost th = TraceyGlobalConfig.getById(hid);
```
Comment (Andrey9kin) [inline src/main/java/org/jenkinsci/tracey/RabbitMQTrigger.java]: variable name th was standing for tracey host now it is completely off scope

File: src/main/java/org/jenkinsci/tracey/RabbitMQTrigger.java [hunk 2]
```diff
@@ -292,21 +276,21 @@ public boolean isApplicable(Item item) {
...
-        public ListBoxModel doFillTraceyHostItems() {
+        public ListBoxModel doFillRabbitMQHostItems() {
             ListBoxModel model = new ListBoxModel();
             TraceyGlobalConfig conf = GlobalConfiguration.all().get(TraceyGlobalConfig.class);
             if(conf != null) {
-                List<TraceyHost> hosts = conf.getConfiguredHosts();
+                List<RabbitMQHost> hosts = conf.getConfiguredHosts();
                 if(hosts != null) {
-                    for(TraceyHost th : hosts) {
+                    for(RabbitMQHost th : hosts) {
```
Comment (Andrey9kin) [inline src/main/java/org/jenkinsci/tracey/RabbitMQTrigger.java]: the same here - th should be replaced with something else

File: src/main/resources/org/jenkinsci/tracey/RabbitMQHost/config.jelly [hunk 3]
```diff
@@ -1,11 +1,11 @@
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout"
          xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:i="jelly:fmt" xmlns:local="local" xmlns:c="/lib/credentials">
-    <f:entry field="description" title="Host description">
+    <f:entry field="description" title="Host name">
         <f:textbox/>
     </f:entry>
 
-    <f:entry field="host" title="Host">
+    <f:entry field="host" title="Path">
```
Comment (Andrey9kin) [inline src/main/resources/org/jenkinsci/tracey/RabbitMQHost/config.jelly]: Is it a host address? Path is super misleading

File: src/main/resources/org/jenkinsci/tracey/RabbitMQHost/help-host.html [hunk 4]
```diff
@@ -0,0 +1,3 @@
+<div>
+    The path to RabbitMQ host
```
Comment (Andrey9kin) [inline src/main/resources/org/jenkinsci/tracey/RabbitMQHost/help-host.html]: Commonly used word is address or URL

File: src/main/resources/org/jenkinsci/tracey/RabbitMQTrigger/help-regexToEnv.html [hunk 5]
```diff
@@ -4,5 +4,6 @@
         <li>Each line represents an extraction</li>
         <li>Each line must conform to the rule of KEY REGEX. There must be a space between the extraction key, and the regex. I.e <em>MYVALUE car=(\S+)</em></li>
         <li>We capture the value of the <em>FIRST</em> capture group in the regex.</li>
+        <li>I.e. BRANCH_FROM_MESSAGE \"branch":\\s+\"([^"']+)\"</li>
```
Comment (Andrey9kin) [inline src/main/resources/org/jenkinsci/tracey/RabbitMQTrigger/help-regexToEnv.html]: You have to show a piece of the message to show from where you are about to extract this info.
Also, mention that two env variables will be injected
BRANCH_FROM_MESSAGE - will contain the whole match, i.e. branch = master
BRANCH_FROM_MESSAGE_1 - will contain first match group, i.e. master

File: src/main/resources/org/jenkinsci/tracey/filter/TraceyPayloadRegexFilter/help-regex.html [hunk 6]
```diff
@@ -0,0 +1,3 @@
+<div>
+    The regex expression to filter messages. I.e. "\s*SomeText" or "[A-Z]{2,7}"
```
Comment (Andrey9kin) [inline src/main/resources/org/jenkinsci/tracey/filter/TraceyPayloadRegexFilter/help-regex.html]: Show an example of the message. It is fine to use Eiffel examples here

File: src/test/java/org/jenkinsci/tracey/JenkinsTriggerTest.java [hunk 7]
```diff
@@ -69,8 +69,8 @@ public void givenFreeStyleJob_ShouldTrigger() throws Exception {
         UsernamePasswordCredentialsImpl creds = createCredentials();
         FreeStyleProject p = createAndConfigureProject(r, "Test_Trigger", creds);
 
-        TraceyTrigger tt = new TraceyTrigger(exchange, creds.getId(), false, false, envKey, Collections.EMPTY_LIST);
-        p.getTriggers().put(new TraceyTrigger.TraceyTriggerDescriptor(), tt);
+        RabbitMQTrigger tt = new RabbitMQTrigger(exchange, creds.getId(), false, envKey, Collections.EMPTY_LIST);
```
Comment (Andrey9kin) [inline src/test/java/org/jenkinsci/tracey/JenkinsTriggerTest.java]: variable name tt doesn't correspond to the naming of the class

**Comment by alexsedova** [comment]
README update is comming

**Pull request merged**